### ISSUE TITLE
A lot of "unsupported format character 'B'" errors in the log

### ISSUE THREAD

**Issue by vykulakov** [alerta/alerta#857]
**Describe the bug**
I have configured AlertManager to send alerts to Alerta with webhook and now I see lot of "unsupported format character 'B'" errors in the Alerta log while all Alerta features work fine. I suppose this error appears on each webhook call:
```
Feb 27 20:24:20 localhost alertad[18197]: --- Logging error ---
Feb 27 20:24:20 localhost alertad[18197]: Traceback (most recent call last):
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/lib/python3.5/logging/__init__.py", line 981, in emit
Feb 27 20:24:20 localhost alertad[18197]:     msg = self.format(record)
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/lib/python3.5/logging/__init__.py", line 831, in format
Feb 27 20:24:20 localhost alertad[18197]:     return fmt.format(record)
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/lib/python3.5/logging/__init__.py", line 568, in format
Feb 27 20:24:20 localhost alertad[18197]:     record.message = record.getMessage()
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/lib/python3.5/logging/__init__.py", line 331, in getMessage
Feb 27 20:24:20 localhost alertad[18197]:     msg = msg % self.args
Feb 27 20:24:20 localhost alertad[18197]: ValueError: unsupported format character 'B' (0x42) at index 856
Feb 27 20:24:20 localhost alertad[18197]: Call stack:
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/lib/python3.5/threading.py", line 882, in _bootstrap
Feb 27 20:24:20 localhost alertad[18197]:     self._bootstrap_inner()
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/lib/python3.5/threading.py", line 914, in _bootstrap_inner
Feb 27 20:24:20 localhost alertad[18197]:     self.run()
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/lib/python3.5/threading.py", line 862, in run
Feb 27 20:24:20 localhost alertad[18197]:     self._target(*self._args, **self._kwargs)
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/lib/python3.5/socketserver.py", line 625, in process_request_thread
Feb 27 20:24:20 localhost alertad[18197]:     self.finish_request(request, client_address)
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
Feb 27 20:24:20 localhost alertad[18197]:     self.RequestHandlerClass(request, client_address, self)
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/lib/python3.5/socketserver.py", line 681, in __init__
Feb 27 20:24:20 localhost alertad[18197]:     self.handle()
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/local/lib/python3.5/dist-packages/werkzeug/serving.py", line 293, in handle
Feb 27 20:24:20 localhost alertad[18197]:     rv = BaseHTTPRequestHandler.handle(self)
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/lib/python3.5/http/server.py", line 422, in handle
Feb 27 20:24:20 localhost alertad[18197]:     self.handle_one_request()
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/local/lib/python3.5/dist-packages/werkzeug/serving.py", line 328, in handle_one_request
Feb 27 20:24:20 localhost alertad[18197]:     return self.run_wsgi()
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/local/lib/python3.5/dist-packages/werkzeug/serving.py", line 270, in run_wsgi
Feb 27 20:24:20 localhost alertad[18197]:     execute(self.server.app)
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/local/lib/python3.5/dist-packages/werkzeug/serving.py", line 258, in execute
Feb 27 20:24:20 localhost alertad[18197]:     application_iter = app(environ, start_response)
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/local/lib/python3.5/dist-packages/flask/cli.py", line 324, in __call__
Feb 27 20:24:20 localhost alertad[18197]:     return self._app(environ, start_response)
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 2309, in __call__
Feb 27 20:24:20 localhost alertad[18197]:     return self.wsgi_app(environ, start_response)
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/local/lib/python3.5/dist-packages/raven/middleware.py", line 100, in __call__
Feb 27 20:24:20 localhost alertad[18197]:     iterable = self.application(environ, start_response)
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/local/lib/python3.5/dist-packages/werkzeug/contrib/fixers.py", line 152, in __call__
Feb 27 20:24:20 localhost alertad[18197]:     return self.app(environ, start_response)
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 2292, in wsgi_app
Feb 27 20:24:20 localhost alertad[18197]:     response = self.full_dispatch_request()
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 1811, in full_dispatch_request
Feb 27 20:24:20 localhost alertad[18197]:     rv = self.preprocess_request()
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 2087, in preprocess_request
Feb 27 20:24:20 localhost alertad[18197]:     rv = func()
Feb 27 20:24:20 localhost alertad[18197]:   File "/usr/local/lib/python3.5/dist-packages/alerta/webhooks/__init__.py", line 24, in before_request
Feb 27 20:24:20 localhost alertad[18197]:     request.get_data().decode('utf-8')
Feb 27 20:24:20 localhost alertad[18197]: Message: 'Webhook Request:\nPOST http://localhost:5000/webhooks/prometheus?api-key=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\n\nUser-Agent: Alertmanager/0.16.0\r\nContent-Length: 1374\r\nContent-Type: application/json\r\nHost: localhost:5000\r\n\r\n{private data here}\n'
Feb 27 20:24:20 localhost alertad[18197]: Arguments: ImmutableMultiDict([('api-key', 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')])
```
**To Reproduce**
Steps to reproduce the behaviour:
1. Configure AlertManager to send alerts to Alerta with webhook
2. Generate some alerts
3. Open Alerta log (I use `journalctl` to see logs)

**Expected behaviour**
I expect no errors in the log

**Environment:**
 - OS: Debian 9.7
 - Version:
    - alerta 6.7.3
    - alerta client 6.5.0
    - requests 2.12.4
    - click 7.0

**Additional context**
Alertad config file:
```
$ cat /etc/alertad.conf
DEBUG = False
USE_PROXYFIX = True
ALLOWED_ENVIRONMENTS = ['Production', 'Development', 'Testing']

LOG_HANDLERS = ['console']
LOG_FILE = '/var/log/alertad.log'
LOG_MAX_BYTES = 5*1024*1024  # 5 MB
LOG_BACKUP_COUNT = 2
LOG_FORMAT = 'simple'

DATABASE_URL = 'postgresql://correct_url'
DATABASE_NAME = 'alerta'

CORS_ORIGINS = [
    'correct_origin'
]

ADMIN_USERS = ['admin']

AUTH_REQUIRED = True
AUTH_PROVIDER = 'gitlab'
GITLAB_URL = 'gitlab_url'
OAUTH2_CLIENT_ID = 'client_id'
OAUTH2_CLIENT_SECRET = 'client_secret'
ALLOWED_GITLAB_GROUPS = ['correct_group']

ALERT_TIMEOUT = 300
```

**Comment by satterly** [comment]
This should have been [fixed in version 6.7.5](https://github.com/alerta/alerta/pull/845) which was released just over a week ago. Can you test this version and let me know if it fixes the issue? Thanks.

**Comment by vykulakov** [comment]
All works fine after updating to 6.7.5. Thanks.
